client/js/gdpr/chart.js

```diff
@@ -10,6 +10,8 @@ export class GdprChart extends HTMLElement {
 
     this.attachShadow({mode: 'open'});
 
+    this.zoom = 1;
+
```

Comment:
add a comment here describing what this.zoom does, and how it only affects horizontal elements